atlasdb-dbkvs/src/test/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/OverflowSequenceSupplierTest.java

```diff
@@ -65,7 +65,7 @@ public void shouldSkipValuesReservedByOtherSupplier() throws Exception {
         final SqlConnection sqlConnection = mock(SqlConnection.class);
 
         when(conns.get()).thenReturn(sqlConnection);
-        when(sqlConnection.selectIntegerUnregisteredQuery(anyString())).thenReturn(1, 1001, 2001);
+        when(sqlConnection.selectLongUnregisteredQuery(anyString())).thenReturn(1L, 1001L, 2001L);
```

Comment:
Fine doing this in a follow up PR, but can we improve the test cases around sequences > Integer.MAX_VALUE